packages/ihe-gateway/server/CodeTemplates/Common Library/Check if document exists in AWS S3 Bucket /Check if document exists in AWS S3 Bucket .js

```diff
@@ -0,0 +1,22 @@
+/**
+	Checks if the XCA ITI-39 document exists in s3
+
+	@param {String} path - the file path
+	@return {Boolean} return boolean
+*/
+function xcaDoesDocumentExist(path) {
```

Comment:
Prob just following the previous pattern, but AFAIK there's no reason to associate this function to XCA, right?

In fact, I'd suggest we implement it using the same pattern we used for Core's `getFileInfoFromS3()`, or at least call it something more generic, like `fileExistsOnS3()`